vm/devices/storage/scsidisk/src/lib.rs

```diff
@@ -1069,6 +1070,7 @@ impl SimpleScsiDisk {
                             },
                             DiskError::InvalidInput
                             | DiskError::MemoryAccess(_)
+                            | DiskError::Hv(_)
```

Comment:
I think this is handled above, right?

Reply:
Yes, that's why it's unreachable!(). 
![image](https://github.com/user-attachments/assets/d7957eef-9f7c-4a89-8433-e86d95b5f74e)

Reply:
oh, I get it - all matches are unreachable. Then why have them here? (sorry for the n00b question)